r/tests/testthat/test-dataset-write.R

```diff
@@ -505,3 +505,145 @@ test_that("Max partitions fails with non-integer values and less than required p
     "max_partitions must be a positive, non-missing integer"
   )
 })
+
+get_num_of_files = function(dir, format) {
+    files <-  list.files(dir, pattern = paste('.', format, sep=""), 
+    all.files = FALSE, recursive = TRUE, full.names = TRUE)
+    length(files)
+}
+
+test_that("Dataset write max open files", {
+  skip_if_not_available("parquet") 
+  # test default partitioning
+  dst_dir <- make_temp_dir()
+  file_format <- "parquet"
+  partitioning <- c("c2")
+  num_of_unique_c2_groups = 5
+
+  record_batch_1 <- record_batch(c1=c(1, 2, 3, 4, 0, 10),
```

Comment:
This is minor, but typically in R we put spaces around `=`, so `c1=c(1, 2, 3, 4, 0, 10)` should be `c1 = c(1, 2, 3, 4, 0, 10)`